frontend/src/hooks/usePlausibleTracker.ts

```diff
@@ -73,7 +73,7 @@ export type CustomEvents =
     | 'order-environments'
     | 'project-navigation'
     | 'productivity-report'
-    | 'release-plans';
+    | 'release-management';
```

Comment:
I assume we don't care about any events registered so far, so I took the liberty to rename this for clarity.

Reply:
Great!